Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,18 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+//Answer/Task-1
+console.log("Gryffindor House Members:");
+hogwarts.forEach(({firstName, lastName, house}) =>{
```

Comment:
Please, fix code formatting on this line as well: not =>{ but => {

Reply:
Hi, Andrei, I have fixed the code formatting.